### DISCUSSION TITLE
[咨询] 这是什么以骸？怎么没见过

### DISCUSSION THREAD

**Discussion by ZMJ2333** [share121/inter-knot#1360] [其他]
![d640ffee11e012b5c212141fab5807ba409fd2b6](https://github.com/user-attachments/assets/5271af7a-ed19-43d0-a1da-20669d83b0d2)
我在 _克里特空洞_ 遇到这么一个……骷髅？
当时我很警觉，意识到背后有人，就打了一下。没想到进入了一个像素的界面。
脑袋里还回响着：

> *你看到了sans，这使你充满了决心

决心？决心是什么？
它说的话类似于 很低沉的“呃呃呃”的机械音，但我竟然能听得懂！（虽然我们正常人都能听懂邦布的“嗯呢”吧…）
更加神奇的是，我对它的攻击变成了……不知道怎么形容，但它好像很灵活，能躲掉我的每次攻击
至于最后……我发现它好像是一个人？所以就选择了 ***仁慈*** 
一通解释后才走掉，它的攻击是迄今目前唯一能让我汗流浃背的……

**Comment by ZMJ2333** [comment]
它好像不是躲……那种方式……类似瞬移！很神奇的生物（不过最后知道是误会后它貌似很友好的还给我了一份“炸雪”（它是这么说的），我没敢吃）

**Comment by su2569** [comment]
# nb

**Reply by ZMJ2333** [reply]
?

**Comment by RETRN1** [comment]
👈一定是丝路屠夫

**Comment by ZMJ2333** [comment]
司马屠夫好像不长这样吧……比这大多了

**Comment by FFT66** [comment]
楼主什么身份啊？

**Comment by zzzuser46** [comment]
左眼没冒鬼火，五星差评！！！

**Comment by ZMJ2333** [comment]
回6楼，这个以骸还有这种形态吗？还好我没有彻底惹他（这里用 _他_ 应该尊敬些）生气……

**Comment by ZMJ2333** [comment]
回5楼，普普通通接了个委托，那位掉后室（一种非线性空间结构）了，我那 ~很古早的~ 萝卜显示 **克里特空洞** 中有个入口。但是一过去发现以太活性高得离谱，入口那堵了好大一块结晶，正想办法就遇到这位骷髅了。
